xalia/Gudl/GudlTokenizer.cs

```diff
@@ -57,6 +57,11 @@ public static TextParser<char> HexChar(int maxdigits)
             from dec in Numerics.Integer.OptionalOrDefault(TextSpan.Empty)
             select double.Parse($"{whole}.{dec}");
 
+        public static TextParser<int> GudlHexConstant =
+            from prefix in Span.EqualTo("0x")
+            from digits in Numerics.HexDigitsUInt32
+            select unchecked((int)digits);

```

Comment:
Anything >= 0x80000000 would become negative. Since I don't have any kind of "unsigned int" value, INT_MIN would otherwise be unexpressable in this way.

Ideally, integers should probably not have a range limit, like in Python. I assumed I'd have to bring in an external dependency for this, which didn't seem worth it, but apparently there is a standard library class: https://learn.microsoft.com/en-us/dotnet/api/system.numerics.biginteger?view=netframework-4.8.1

I'll switch over to this, assuming no unanticipated problems.